app/users/models.py

```diff
@@ -1,7 +1,8 @@
 from app import db
+from app.lib.models import Base
 
-class User(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
+
+class User(Base):
```

Comment:
Similar comments about using the declared base, your mixin, and adding doc strings.